.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ "3.8" ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        run: |
+          sudo apt-get install python3
+      - name: Install dependencies
+        run: |
+          sudo apt-get install build-essential  python3-dev
+          python3 -m pip install --upgrade pip
+          pip3 install pylint
+          pip3 install pytest pytest-cov pytest-subtests
+          pip3 install coveralls
+          pip3 install -r requirements.txt
+      - name: Setup
+        run: |
+          pip install -e .
+      - name: Running pytest
+        run: |
+          pytest cluster_generator --doctest-modules --answer_dir="./tmp" --answer_store --gravity="Newtonian" --cov=cluster_generator --cov-report=xml
+          COVERALLS_REPO_TOKEN=tQ0ra8kUn56CDx6Cjp7Wil8Pp5qA9vTdK coveralls
```

Comment:
@Eliza-Diggins I assume that this is your token? at some point we'd want to switch to mine at some point.

Reply:
@jzuhone Yes! I've actually completely removed this in the next pull request and replaced it with a remote host and answer fetch approach like the one you have set up on pyxsim. We will still need to fiddle with some bits to host the answers somewhere suitable and what not, but we can figure that out in a later pull request.